app/invocation/invocation_timing_card.tsx

```diff
@@ -140,6 +142,12 @@ export default class InvocationTimingCardComponent extends React.Component<Props
         } else {
           this.state.durationMap.set(event.name, event.dur);
         }
+
+        if (this.state.durationByCategoryMap.get(event.cat)) {
```

Comment:
i think if you wanna make this a one-liner, you can do...

`set(event.cat, (this.state.durationByCategoryMap.get(event.cat) || 0) + event.dur)`

side note, this usage of a map in state is.. unusual and the map might as well be a regular member of the class (updating it this way wouldn't cause a re-render), but that's unrelated to this change.

Reply:
Done.